tests/cases/standalone/common/copy/copy_database_from_fs_parquet.result

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE demo(host string, cpu double, memory double, ts TIMESTAMP time index);
+
+Affected Rows: 0
+
+INSERT INTO demo(host, cpu, memory, ts) values ('host1', 66.6, 1024, 1655276557000), ('host2', 88.8,  333.3, 1655276558000);
+
+Affected Rows: 2
+
+COPY DATABASE public TO '/tmp/demo/export/parquet/' WITH (FORMAT="parquet");
```

Comment:
We'd better add a case that contains the time range filter.

Reply:
> We'd better add a case that contains the time range filter.

added here https://github.com/GreptimeTeam/greptimedb/commit/1da7d2e6b7e02f972b92d52e23d821af060a1361#diff-efbd2026da9f11c13a56a8bee19745fab9f054aa9e2a65601615c16901cd71c1R17-R22